### DISCUSSION TITLE
Enable web search via URL

### DISCUSSION THREAD

**Discussion by jsuelwald** [open-webui/open-webui#5690] [General]
Hi :)

i know that `https://<your-open-webui-url>/?q=%s `already exist.

Now i want to automatically enable "Web Search" via the URL as well. Is that possible?

My usecase is the following: I want to add my instance of open-webui as search engine (which works very well, thanks for the good documentation), but for those search reqeusts i want open-webui do a web search for those queries.

**Comment by jsuelwald** [comment]
Also.. it would be nice to have a list which URL parameters (like `model`) exist, but i didn't find a list in the documentation.

**Comment by Mavyre** [comment]
You can add `web-search=true` in your url parameters and it'll automatically ask with web search enabled. Couldn't find a list of URL parameters in the docs either though...
It looks like `https://<your-open-webui-url>/?q=%s&web-search=true` on mine and works perfectly
